src/libevent/SConscript

```diff
@@ -30,7 +32,11 @@ def CheckSizeOf(ctx, type, defaults, includes = None):
 
     ctx.Message('Checking size of %s ... ' % type)
 
-    for sz in defaults + range(1, 16):
+    sizes = range(1, 16)
+    # In Python 3, explicit conversion to the list required
+    if sys.version_info[0] >= 3:
+        sizes = list(sizes)
+    for sz in defaults + sizes:
```

Comment:
I would be simpler to write it like this:

    for sz in defaults + list(range(1, 16)):

Reply:
You're right. I've just submitted the fix.